.circleci/config.yml

```diff
@@ -26,6 +26,22 @@ jobs:
       - run: cp /tmp/builds/libgo_cosmwasm.so ./api
       - run: make test
       - run: make build-go
+  # TODO: only run this on master (and not on tags)
+  release:
+    docker:
+      - image: circleci/rust:1.41-stretch
+    steps:
+      - run: rustup toolchain install nightly
+      - checkout
+      - run: make release
+      - run: ls -l ./api
+      # TODO: check if there is a diff (only commit then)
```

Comment:
Side note: without `--allow-empty`, `git commit` fails anyways if there is no diff staged.

Reply:
Actually I realize the .so files will *always* be different, cuz they are not run in deterministic conditions (compile with 16 threads). So, there should always be a diff. I will add the flag just in case